docs/BYO_INFRASTRUCTURE.adoc

```diff
@@ -0,0 +1,60 @@
+= Using the CASL Tools with Static Infrastructure (Bring Your Own Infrastructure)
+
+This guide walks through the process of using the tools in this repository to automate the stand up of OpenShift clusters on Infrastructure where there is currently no automated provisioner. This process can easily be used in on-premise installs where the infrastructure is not currently API driven and has yet to be automated in a way that's compatible with CASL.
+
+== What Infrastructure do I need before I run CASL?
+
+If you read through the link:PROVISIONING_ARCH.md[CASL Provisioning Architecture documentation], you'll notice the following description of our _Provisioning Phase_:
+
+_Provision Infrastructure_ creates the servers/instances and any other supporting infrastructure required (network, DNS, load balancers, etc.). In general this phase
+  - Begins with having credentials with proper access to an Infrastructure provider.
+  - Ends with servers provisioned and running with an OS installed and SSH keys (with root or sudo) pushed.
```

Comment:
May want to include some DNS info here + proper cert handling (especially when LBs are in-use)